### PR TITLE
[Textfield] auto size height

### LINKED ISSUE #627: [Textfield] auto size height
It was quite hard to plug CDK auto height, so we should add it to LF 

https://material.angular.io/cdk/text-field/overview

### PR DESCRIPTION
**Pull request by almonais**
fixes #627 

image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://846-pr-lucca-front-LuccaSA.surge.sh

File: packages/ng/libraries/core/src/style/components/_input.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
 @import "~@lucca-front/scss/src/mixins/forms";
+@import "@angular/cdk/text-field-prebuilt.css";
```
Comment (lucienbertin) [inline packages/ng/libraries/core/src/style/components/_input.scss]: we dont want to import angular/cdk css

if you look at the style for the overlay [here](https://github.com/LuccaSA/lucca-front/blob/master/packages/ng/libraries/core/src/style/components/_overlay.scss) you'll see that we dont import their css but we use their selectors

File: packages/ng/libraries/core/src/style/components/_input.scss [hunk 2]
```diff
@@ -13,3 +15,10 @@
 		@include fieldFramedError("textfield");
 	}
 }
+
+.cdk-textarea-autosize {
```
Comment (lucienbertin) [inline packages/ng/libraries/core/src/style/components/_input.scss]: i'd rather you created a new file to add this code

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://846-pr-lucca-front-LuccaSA.surge.sh

File: packages/ng/libraries/core/src/style/components/_input.scss (hunk 1 above)
Comment (GuillaumeNury) [reply packages/ng/libraries/core/src/style/components/_input.scss]: du coup, il faut intégrer le contenu de `@angular/cdk/text-field-prebuilt.css` dans LF ou charge aux BUs d'importer ce fichier ?

Comment (lucienbertin) [reply packages/ng/libraries/core/src/style/components/_input.scss]: english

the target is that all will be handled by LF, no need to import another third party stylesheet

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://846-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-846

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-846

**Pull request merged**